projectroles/forms.py

```diff
@@ -957,6 +957,24 @@ def clean(self):
         except ProjectInvite.DoesNotExist:
             pass
 
+        # Local users check
+        user_email = self.cleaned_data.get('email')
+        domain = user_email[user_email.find('@') + 1 :]
+        domain_list = [
+            getattr(settings, 'AUTH_LDAP_USERNAME_DOMAIN', ''),
+            getattr(settings, 'AUTH_LDAP2_USERNAME_DOMAIN', ''),
+        ]
+        if (
+            not settings.PROJECTROLES_ALLOW_LOCAL_USERS
+            and not settings.ENABLE_SAML
+            and domain not in domain_list
+        ):
```

Comment:
We are missing the check for `LDAP_ALT_DOMAINS` here. That also must be included as stated in the original spec.

**Note:** While `AUTH_LDAP*_USERNAME_DOMAIN` is in the form of e.g. `CHARITE` or `MDC-BERLIN` (without the top level domain), `LDAP_ALT_DOMAINS` is a list with TLDs, e.g. `['bih-charite.de']`.

Also, use `.lower()` on each value before comparison, as `AUTH_LDAP*_USERNAME_DOMAIN` values are expected as upper case.